app/templates/views/guidance/index.html

```diff
@@ -10,24 +10,24 @@
 
   <h1 class="heading-large">Guidance</h1>
 
-  <p class="govuk-body">This guidance is for teams using U.S. Notify to send emails and text messages.</p>
+  <p class="govuk-body">This guidance is for teams using U.S. Notify to send text messages.</p>
 
   <p class="govuk-body">It explains how to:</p>
 
   <ul class="list list-bullet">
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.edit_and_format_messages') }}">edit and format messages</a></li>
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.branding_and_customisation') }}">add branding and customization</a></li>
-    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.send_files_by_email') }}">send files by email</a></li>
+   <!-- <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.send_files_by_email') }}">send files by email</a></li> -->
   </ul>
 
   <h2 class="heading-medium">More information</h2>
```

Comment:
[suggestion] this heading should be included in the comment-out. otherwise it's just hanging out with no content

Reply:
@tdlowden it doesn't look like anything has been done with this header that doesn't actually have any content.

Reply:
Aha. ok my bad. think i got it now